react/components/RequestForm.tsx

```diff
@@ -734,4 +850,23 @@ class RequestForm extends Component<Props> {
   }
 }
 
-export default injectIntl(RequestForm)
+const compose = (...funcs) =>
+  funcs.reduce(
+    (a, b) =>
+      (...args) =>
+        a(b(...args)),
+    (arg) => arg
+  )
+
+export default compose(
+  graphql(NEAR_PICKUP_POINTS, {
+    options: () => ({
+      variables: {
+        lat,
+        long,
+      },
+      ssr: false,
+    }),
+    name: 'NearestPickupPoints',
```

Comment:
What do you think of changing this name?
Or maybe adding an `alias` to the query on the .gql file, so we don't repeat twice `NearestPickupPoints`